server/routes.js

```diff
@@ -51,6 +56,25 @@ module.exports = (expressApp, nextApp) => {
     return res.sendFile(path.join(__dirname, '../public/static/images/favicon.ico.png'));
   });
 
+  // Helper to enable downloading files that are on S3 (on another origin)
+  app.get('/download-file', async (req, res) => {
```

Comment:
I'm curious about why we need that?

Reply:
Chrome and Firefox does not allow cross-origin downloads when using the `download` attribute on an anchor tag, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download.

Reply:
Interesting, it would be great to mention it in the route comment.

What we're doing here could be a first step towards https://github.com/opencollective/opencollective/issues/6351.

Reply:
Added a better comment!